include/net/aws_iot.h

```diff
@@ -104,18 +90,24 @@ enum aws_iot_evt_type {
 	 *  needs to be reinitialized to apply the new modem image.
 	 */
 	AWS_IOT_EVT_FOTA_DONE,
+
 	/** FOTA erase pending. */
 	AWS_IOT_EVT_FOTA_ERASE_PENDING,
+
 	/** FOTA erase done. */
 	AWS_IOT_EVT_FOTA_ERASE_DONE,
-	/** FOTA progress notification. */
-	AWS_IOT_EVT_FOTA_DL_PROGRESS,
-	/** FOTA error. Used to propagate FOTA-related errors to the
-	 *  application. This is to distinguish between AWS_IOT irrecoverable
-	 *  errors and FOTA errors, so they can be handled differently.
+
+	/** FOTA progress notification.
+	 *  Payload is of type int (.data.fota_progress)
 	 */
+	AWS_IOT_EVT_FOTA_DL_PROGRESS,
+
+	/** FOTA error. Something went wrong during the FOTA process, try again. */
 	AWS_IOT_EVT_FOTA_ERROR,
-	/** AWS IoT library irrecoverable error. */
+
+	/** Irrecoverable error, if this event is received the device should perform a reboot.
```

Comment:
Is it always that severe if this error is seen?

Reply:
Well, not always. Room for improvement here.